mods/core/src/test/usermanager.unit.test.ts

```diff
@@ -31,4 +35,60 @@ describe('@fonos/core/usermanager', () => {
     expect(roleStub).to.has.been.calledOnce
     expect(hasAccess).to.be.equal(false)
   })
+
+
+})
+
+
+describe('@fonos/core/usermanager/userOperations', () => {
+
+  const sampleUser = new User({
+    firstName : "test",
+    lastName : "test",
+    email:"test@gmail.com",
+    role : "TEST",
+    accessKeyId :"70lkv20G5MCgUIKYJI6Z",
+    createTime : "2020-10-10",
+    updateTime : "2020-10-10",
+    status : "ACTIVE"
+   });
+
+  afterEach(() => sandbox.restore())
+
+  context('teting userOperation', () => {
+
+  it('should retrieve all users', async () => {
+   const findAllStub = sinon.stub(UserController.prototype,"getAll").returns(Promise.resolve([sampleUser]));
+   const user = new UserController();
+   user.getAll();
+   expect(findAllStub).to.has.been.calledOnce;
+   expect(sampleUser).to.be.a('object');
```

Comment:
You should compare the input with the return. For example, if the property "firstName" is part of the input and output, you should compare them.